docker-compose.yml

```diff
@@ -7,12 +7,20 @@ services:
     restart: unless-stopped
     network_mode: host
     environment:
+      - PUID=508
+      - PGID=508
+
       - MANAGE_HTTP_PORT=8088
       - MANAGE_HTTPS_PORT=8043
```

Comment:
Should be a default of `PORTAL_HTTPS_PORT=8843`; this is actually an existing error in the compose file.

Reply:
True, fixed!